kedro-datasets/tests/pandas/test_deltatable_dataset.py

```diff
@@ -66,7 +66,7 @@ def test_append(self, deltatable_dataset_from_path, dummy_df):
         """Test saving by appending new data."""
         deltatable_dataset_from_path.save(dummy_df)
         new_df = pd.DataFrame({"col1": [0, 0], "col2": [1, 1], "col3": [2, 2]})
-        appended = pd.concat([dummy_df, new_df], ignore_index=True)
+        appended = pd.concat([new_df, dummy_df], ignore_index=True)
```

Comment:
Unsure why with the new pyarrow version the append behaviour changes and if this breaks the expected behaviour

Reply:
This seems to be a bug and it should have a fix instead of changing the test.